simulation/collectors.py

```diff
@@ -129,10 +129,6 @@ def household_annual_kwh_heating_demand(household) -> int:
     return int(household.annual_kwh_heating_demand)
 
 
-def household_annual_heating_fuel_bill(household) -> int:
-    return household.annual_heating_fuel_bill
```

Comment:
I guess we are removing this since now the input depends on the model.

Should we add a `@on_first_step(model_price_gbp_per_kwh_{gas|electricity|oil})` collector to the `model_collectors` to enable us to calculate the fuel bills?

Or should we not save it to the output dataframe and instead calculate it as and when needed?